packages/admin/src/table/filterbar/filterBarButton/FilterBarButton.styles.tsx

```diff
@@ -0,0 +1,60 @@
+import { createStyles, Theme } from "@material-ui/core";
+
+import { FilterBarButtonProps } from "./FilterBarButton";
+
+export type FilterBarButtonClassKey = "root" | "open" | "hasDirtyFields" | "filterBadge";
+
+export const styles = (theme: Theme) => {
+    return createStyles<FilterBarButtonClassKey, FilterBarButtonProps>({
+        root: {
+            position: "relative",
+            alignItems: "center",
+            padding: "10px 15px",
+            cursor: "pointer",
+            display: "flex",
+            border: `1px solid ${theme.palette.grey[100]}`,
+            textTransform: "initial",
```

Comment:
This does the same thing and is compatible with more browsers 🙂

```suggestion
            textTransform: "none",
```